contracts/protocol/facets/ProtocolInitializationHandlerFacet.sol

```diff
@@ -122,7 +122,6 @@ contract ProtocolInitializationHandlerFacet is IBosonProtocolInitializationHandl
         uint256 _maxPremintedVouchers = abi.decode(_initializationData, (uint256));
         require(_maxPremintedVouchers != 0, VALUE_ZERO_NOT_ALLOWED);
         protocolLimits().maxPremintedVouchers = _maxPremintedVouchers;
-        emit MaxPremintedVouchersChanged(_maxPremintedVouchers, msgSender());
```

Comment:
I think this still needs a proper resolution since this `initV2_2_0` is currently not doing what is expected from `initV2_2_0`.

Either we should remove "old" inits. 
Or probably simpler, just keep the event `MaxPremintedVouchersChanged` in `IBosonConfigEvents`, and keep all inits so we have "history" in one place. (I'm more for this option).